lib/salus/scanners/yarn_audit.rb

```diff
@@ -127,6 +133,91 @@ def handle_legacy_yarn_audit
       report_failure
     end
 
+    def auto_fix_vulns
```

Comment:
It would be safer to wrap the entire `auto_fix_vulns` in a `begin ... rescue ...`, just in case we get unexpected values.